### PR TITLE
input: add base dark

### LINKED ISSUE #190: Add Dark base in Input component
### Context
Refactor `Input` component to use the Dark base in `Account` pages.

### Example
`Zeplin`: http://zpl.io/VqOdBA7

### Requirements
- [x] Add Base Dark props to input component

### PR DESCRIPTION
**Pull request by oieusouamiguel**
## Context
Refactor `Input` component to use the Dark base.

## Checklist
- [x] Add Base Dark class to `former-kit-skin`
- [x] Add Base Dark props to Input component
- [x] Add Dark and Light bases in Former-kit Storybook

## Linked Issues
- [x] Resolves #190 

### Layout:
`Zeplin`: http://zpl.io/VqOdBA7

### Preview:
image

### CODE REVIEW

File: stories/Input/form.js [hunk 1]
```diff
@@ -148,7 +162,13 @@ storiesOf('Inputs', module)
       </Section>
 
       <Section title="Icon multiline error">
-        <InputState multiline error="Error!" icon={<IconMail width={16} height={16} />} />
+        <InputState
+          multiline
+          error="Error!"
+          icon={
+            <IconMail width={16} height={16} />
+            }
```
Comment (igorantun) [inline stories/Input/form.js]: Shouldn't this line be 2 spaces to the left?

File: stories/Section.js [hunk 2]
```diff
@@ -5,11 +5,17 @@ import classnames from 'classnames'
...
   title,
   children,
   className,
 }) => (
-  <section className={classnames(styles.section, className)}>
+  <section className={
+    classnames(
+      styles[base],
+      styles.section,
+      className)}
```
Comment (igorantun) [inline stories/Section.js]: Shouldn't the `)` be in the line below, and the `}`, two lines below?

File: stories/Section.js [hunk 3]
```diff
@@ -5,11 +5,17 @@ import classnames from 'classnames'
...
+  >
     {title &&
```
Comment (igorantun) [inline stories/Section.js]: Not sure if it needs to be changed, but I would put the `title &&` operator in the line below, with the component. Another option is putting it in a separate line, below, and have the component below it (as long as the opening bracket has its own line)

**Comment by alalbux** [comment]
atualizar a branch com a master @oieusouamiguel

File: src/Input/Input.js [hunk 4]
```diff
@@ -270,6 +273,7 @@ Input.propTypes = {
    */
   theme: PropTypes.shape({
     active: PropTypes.string,
+    base: PropTypes.string,
```
Comment (lucasjs) [inline src/Input/Input.js]: Aqui são passadas as classes de CSS, que no caso, base não é uma. Precisa passar light e dark.

Comment (oieusouamiguel) [reply src/Input/Input.js]: Valeu!

**Comment by uselessdev** [comment]
Seria interessante nos `inputs default` adicionar também exemplos com base dark.

**Pull request merged**